src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  font-family: $main-font-famaly;
+}
+
+.contentInformacion {
+  margin: 0 16 16;
```

Comment:
The 'margin' property values are missing units. Ensure that each value has a unit (e.g., 'px', 'em'). For example, 'margin: 0 16px 16px;'.